### PR TITLE
fix: widget id as anonymous

### PR DESCRIPTION
**Pull request by Yuvalke**
### Description of the Changes

Please add a detailed description of the change, whether it's an enhancement or a bugfix.
If the PR is related to an open issue please link to it.

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: test/src/k-provider/ovp/provider.spec.js [hunk 1]
```diff
@@ -331,13 +331,36 @@ describe('getMediaConfig', function() {
     });
   });
 
+  it('should not be anonymous with widgetId', done => {
```
Comment (OrenMe) [inline test/src/k-provider/ovp/provider.spec.js]:
```suggestion
  it('should set anonymous to false when given a widgetId', done => {
```

File: test/src/k-provider/ovp/provider.spec.js [hunk 2]
```diff
@@ -331,13 +331,36 @@ describe('getMediaConfig', function() {
...
+
   it('should pass _partnerId to the session loader', done => {
     provider = new OVPProvider({partnerId}, playerVersion);
     provider.getMediaConfig({entryId: '1_rwbj3j0a'}).catch(() => {
       provider._dataLoader._loaders.get('session')._widgetId.should.equal('_1068292');
       done();
     });
   });
+
+  it('should send the response ks on request with widgetId', done => {
```
Comment (OrenMe) [inline test/src/k-provider/ovp/provider.spec.js]:
```suggestion
  it('should use the response KS on request with widgetId', done => {
```

File: src/k-provider/ovp/provider.js [hunk 3]
```diff
@@ -98,6 +101,9 @@ export default class OVPProvider extends BaseProvider<OVPProviderMediaInfoObject
         const sessionLoader = data.get(OVPSessionLoader.id);
         if (sessionLoader && sessionLoader.response) {
           mediaConfig.session.ks = sessionLoader.response;
+          if (this.widgetId !== this.defaultWidgetId) {
```
Comment (yairans) [inline src/k-provider/ovp/provider.js]: I would change it to:
```suggestion
          if (!this. _isAnonymous) {
```

File: src/k-provider/ovp/provider.js [hunk 4]
```diff
@@ -184,6 +193,9 @@ export default class OVPProvider extends BaseProvider<OVPProviderMediaInfoObject
       this.ks = entryListInfo.ks;
       this._isAnonymous = false;
     }
+    if (this.widgetId !== this.defaultWidgetId) {
```
Comment (yairans) [inline src/k-provider/ovp/provider.js]: same

File: src/k-provider/ovp/provider.js (hunk 3 above)
Comment (Yuvalke) [reply src/k-provider/ovp/provider.js]: no, I want to change it if you entered widgetId

Comment (Yuvalke) [reply src/k-provider/ovp/provider.js]: It'll be much easier to understand why

**Pull request merged**